src/service_control_client_impl_quota_test.cc

```diff
@@ -260,11 +260,13 @@ TEST_F(ServiceControlClientImplQuotaTest, TestCachedQuotaRefreshGotHTTPError) {
   EXPECT_EQ(quota_response.allocate_errors_size(), 0);
 
   // AllocateQuotaFlushCallback replaces the cached response with the
-  // negative response, QUOTA_EXHAUSED
+  // negative response, QUOTA_EXHAUSTED
 
-  // Set callback function to return the negative status
-  // to simulate HTTP error
-  mock_quota_transport_.done_status_ = CancelledError("");
+  // Set the callback function to return a fail-close negative status, e.g.
+  // CancelledError, to simulate HTTP error. A fail-close negative status
+  // will be cached as a negative response.
+  Status cancelledError = CancelledError("cancelled error");
+  mock_quota_transport_.done_status_ = cancelledError;
 
   // Wait 600ms to trigger the next quota refresh
   std::this_thread::sleep_for(std::chrono::milliseconds(600));
```

Comment:
Could you add comment here.   
// This sleep will not trigger a remote quota call since test did not provide a timer object.
// The refresh remote quota call is triggered by cache lookup in the "Quota" call,  
// for an expired entry, if it has aggregated cost or it is negative, make a remote quota call.